decoder/incident.go

```diff
@@ -118,7 +147,22 @@ func createIncidentWebhooks(ctx context.Context, db db.DbDetails, oldIncident *I
 		if stop == nil {
 			stop = &Pokestop{}
 		}
-
+		lineup := [3]webhookLineup{}
```

Comment:
Simpler syntax for this - see playground https://go.dev/play/p/NFIwWJ4rD8L

Reply:
Resolved in latest, thanks for the playground!